.gitignore

```diff
@@ -1,5 +1,162 @@
-__pycache__
-*.pyc
+# Byte-compiled / optimized / DLL files
```

Comment:
I don't love these "everything" `.gitignore` files, let's just limit it to what we actually need for this project: perhaps all we need is `*.pyc`, `__pycache__/` for now?

Reply:
Reverted!